src/current/v23.2/cluster-virtualization-overview.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Cluster Virtualization Overview
+summary: Learn more about cluster virtualization and how it facilities cluster-to-cluster replication.
+toc: true
+docs_area: deploy
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+
+Refer to the [Known Limitations](#known-limitations) section for further detail.
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} This page gives an overview of _cluster virtualization_ in CockroachDB {{page.version.version}}. Cluster virtualization allows you to separate a cluster's _control plane_ from its _data plane_. The cluster's control plane manages cluster nodes and node-to-node traffic, while its data plane reads data from and writes data to the cluster's storage.
+
+{{site.data.alerts.callout_success}}
+Cluster virtualization is enabled automatically when you configure [Physical Cluster Replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}).
+{{site.data.alerts.end}}
+
+When cluster virtualization is disabled, the control plane and data plane are unified, and the `cockroach` process on a node handles all system and user activity, and manages the cluster's control plane and data plane. When cluster virtualization is enabled, the `cockroach` process on a node runs both a _system virtual cluster_ and one or more _virtual clusters_.
+
+- The system virtual cluster manages the cluster's control plane. Administrative access to the system virtual cluster can be restricted. Certain low-level cluster settings can be modified only on the system virtual cluster.
+- A virtual cluster manages its own data plane. Administrative access on a virtual cluster does not grant any access to the system virtual cluster. The effect of some settings is scoped to the virtual cluster rather than the system virtual cluster.
```

Comment:
This is not something that your PR is causing, but reading through all the pages and  the changes to the PCR docs, the naming is still really awkward to read with "cluster" everywhere. It's almost in every sentence 2–4 times. If the aim is to stick with this naming, do we want to create an acronym/initialism at the start of this page for virtual cluster?

Reply:
I think this should be discussed with the PM and probably doesn't need to be decided right away. WDYT